Kitodo/src/main/webapp/WEB-INF/templates/includes/importConfigurationEdit/rows/searchFieldRows.xhtml

```diff
@@ -80,13 +80,29 @@
                                   noSelectionOption="true"/>
                     <f:selectItems value="#{importConfigurationEditView.searchFields}"/>
                     <p:ajax event="change"
+                            update="editForm:importConfigurationTabView:idParameterPrefix"
                             oncomplete="toggleSave();"/>
                 </p:selectOneMenu>
                 <p:commandButton id="idSearchFieldHelp" type="button"
                                  styleClass="help-button" icon="fa fa-lg fa-question-circle-o"/>
                 <p:tooltip for="idSearchFieldHelp"
                            value="URL parameter for unique identifiers. This parameter is used for importing individual records from the search interface."/>
```

Comment:
Translatable string would be here nice too.

Reply:
Yes, I know. I haven't translated the tooltips for all those `ImportConfiguration` edit form fields, yet. See my description in #5038, I was planning to add the translations for all those tooltips in a separate pull request, probably in a new "tooltip" bundle.

Reply:
See #5235

Reply:
Should be okay if resolving the mentioned issue is done in near future - at least before next release.